code/turf/turf_autoalign.dm

```diff
@@ -486,6 +488,7 @@ ABSTRACT_TYPE(turf/simulated/wall/auto/hedge)
 	light_mod = "wall-"
 	flags = ALWAYS_SOLID_FLUID | IS_PERSPECTIVE_FLUID
 	connect_diagonal = 1
+	default_material = "wood"
```

Comment:
won't this make them a 'wood hedge'

Reply:
also labels

Reply:
Fixed, default material gets setname=FALSE and appearance=FALSE